UserTools/Configuration/Configuration.h

```diff
@@ -45,12 +45,18 @@ class Configuration: public Tool {
 
  private:
 
+  std::string m_configfile;
   static void Thread(Thread_args* arg); ///< Function to be run by the thread in a loop. Make sure not to block in it
   Utilities* m_util;  ///< Pointer to utilities class to help with threading
   Configuration_args* args; ///< thread args (also holds pointer to the thread)
 
   bool LoadConfig();
   
+  // verbosity levels: if 'verbosity' < this level, the message type will be logged.
+  int v_error=0;
```

Comment:
think this would generally be usefull in the logging class of toolframework. think you mentioned this before. but thats where we should define them like colours.

Reply:
i have been adding these to my tools for [over six years](https://github.com/ANNIEsoft/ToolAnalysis/pull/38), asking for this to be part of the framework. :smiling_face_with_tear: please, let's.